components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -452,93 +452,128 @@ private String getOIDCAuthzEndpoint(Map<String, String> authenticatorProperties)
     protected void processAuthenticationResponse(HttpServletRequest request, HttpServletResponse response,
                                                  AuthenticationContext context) throws AuthenticationFailedException {
 
-        try {
+        OAuthClientResponse oAuthResponse = generateOauthResponse(request, context);
 
-            OAuthAuthzResponse authzResponse = OAuthAuthzResponse.oauthCodeAuthzResponse(request);
-            OAuthClientRequest accessTokenRequest = getAccessTokenRequest(context, authzResponse);
+        // TODO : return access token and id token to framework
```

Comment:
Why do we have this `TODO`?

Reply:
That was an old TODO comment, Formatted until proper clarifications